packages/mgt-components/src/components/mgt-todo/mgt-todo.scss

```
@@ -93,12 +92,13 @@
   }
 }
 
-:host fluent-text-field#new-taskName-input.outline::part(start) {
-  margin-inline-start: unset;
+:host fluent-text-field#new-taskName-input.outline::part(start),
+:host fluent-text-field#new-taskName-input.outline::part(end) {
+  margin-inline: unset;
 }
```

@Mnickii for a much _cleaner_ css, I suggest you use blocks of common classes. For example here
```suggestion
:host{
  fluent-text-field {
    &::part(start),
    &::part(end) {
      margin-inline: unset;
    }
  }
}
```
Now you don't have to keep track of the element `ids` or shadow DOM classes.